data/transactions/logic/assembler.go

```diff
@@ -1573,13 +1583,16 @@ func (ops *OpStream) assemble(text string) error {
 				case "pragma":
 					ops.pragma(tokens)
 					ops.trace("%3d: #pragma line\n", ops.sourceLine)
+				case "define":
+					ops.define(tokens)
+					ops.trace("%3d: #define line\n", ops.sourceLine)
 				default:
 					ops.errorf("Unknown directive: %s", directive)
 				}
 				continue
 			}
 		}
-		for current, next := splitTokens(tokens); len(current) > 0 || len(next) > 0; current, next = splitTokens(next) {
+		for current, next := splitTokens(ops, tokens); len(current) > 0 || len(next) > 0; current, next = splitTokens(ops, next) {
```

Comment:
Now that it's more involved than just splitting, maybe `splitTokens` is `nextOperation` or `nextStatement`.